src/controllers/ignores/IgnorePhrase.cpp

```diff
@@ -112,4 +113,11 @@ bool IgnorePhrase::containsEmote() const
     return !this->emotes_.empty();
 }
 
+IgnorePhrase IgnorePhrase::createEmpty()
+{
+    return IgnorePhrase(
+        QString(), false, false,
+        ::chatterino::getSettings()->ignoredPhraseReplace.getValue(), true);
```

Comment:
The big namespace is no longer necessary since we're part of the namespace here

```suggestion
        getSettings()->ignoredPhraseReplace.getValue(), true);
```